keyboards/handwired/mutepad/readme.md

```diff
@@ -0,0 +1,33 @@
+# MutePad
+
+![MutePad Repo](https://github.com/JoshwJB/MutePad)
```

Comment:
This syntax is for an image, not a link.

Reply:
Updated now. Thanks for taking a look.